flow/src/main/java/flow/Flow.java

```diff
@@ -65,6 +65,11 @@
     return wrapper.services.getService(serviceName);
   }
 
+  /** @return null if context does not contain the named service. */
+  @Nullable public static <T> T getService(@NonNull String serviceName, @NonNull View view) {
```

Comment:
I think it's nice.  Nine times out of ten when you're using Flow, the Context you interact with comes from a View.  This is a simple convenience, but it'll get used all over the place.